### PR TITLE
Add modern theme

### PR DESCRIPTION
**Pull request by sebaskleeen**
**Changes**:
- Add modern theme

**Other changes**:
- Fix chart colors
- Fix table colors
- Fix table styles
- Modularize chart configurations
- Fix minor material style inconsistencies

Example images (click them for larger view):
Screen Shot 2020-06-11 at 1 38 44 PMScreen Shot 2020-06-11 at 1 40 00 PMScreen Shot 2020-06-11 at 1 40 00 PMScreen Shot 2020-06-11 at 1 38 44 PM

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/PositiveNegativeArea/PositiveNegativeArea.tsx [hunk 1]
```diff
@@ -1,32 +1,15 @@
...
-  chart: {
-    type: 'area',
-    backgroundColor: 'var(--transparent)',
-  },
-  yAxis: {
-    title: {
-      text: null,
-    },
-  },
+const baseOptions: Highcharts.Options = merge({}, generalBaseOptions, {
```
Comment (aaron-kleeen) [inline libs/atomic-elements/src/lib/components/PositiveNegativeArea/PositiveNegativeArea.tsx:8]: I was wondering if the empty `{}` is necessary
```suggestion
const baseOptions: Highcharts.Options = merge(generalBaseOptions, {
```

File: libs/atomic-elements/src/lib/components/Scatter/Scatter.tsx [hunk 2]
```diff
@@ -1,78 +1,14 @@
...
-        },
-      },
-      tooltip: {
-        headerFormat: '',
-        pointFormat: '{point.category}: {point.y}',
-      },
-    },
-  },
-};
+const baseOptions: Highcharts.Options = merge({}, generalBaseOptions, {
```
Comment (aaron-kleeen) [inline libs/atomic-elements/src/lib/components/Scatter/Scatter.tsx:8]:
```suggestion
const baseOptions: Highcharts.Options = merge(generalBaseOptions, {
```

File: libs/atomic-elements/src/lib/components/generalBaseOptions.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  plotOptions: {
+    area: {
+      lineWidth: 1,
+    },
+    pie: {
+      allowPointSelect: true,
+      borderWidth: 0,
+      cursor: 'pointer',
+      dataLabels: {
+        // enabled: false,
```
Comment (aaron-kleeen) [inline libs/atomic-elements/src/lib/components/generalBaseOptions.ts:60]: is this necessary?

File: libs/atomic-elements/src/lib/components/generalBaseOptions.ts [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        color: 'var(--on-surface-color)',
+        style: {
+          textOutline: '0',
+        },
+      },
+      // showInLegend: true,
```
Comment (aaron-kleeen) [inline libs/atomic-elements/src/lib/components/generalBaseOptions.ts:66]: what about this?

File: libs/atomic-elements/src/lib/components/PositiveNegativeArea/PositiveNegativeArea.tsx (hunk 1 above)
Comment (sebaskleeen) [reply libs/atomic-elements/src/lib/components/PositiveNegativeArea/PositiveNegativeArea.tsx:8]: it is, I teste without initially, but merge modifies the reference, and doesnt create a new object everytime

File: libs/atomic-elements/src/lib/components/generalBaseOptions.ts (hunk 3 above)
Comment (sebaskleeen) [reply libs/atomic-elements/src/lib/components/generalBaseOptions.ts:60]: whoops, yes it is let me fix that

**Pull request merged**